packages/@expo/cli/src/start/server/metro/withMetroMultiPlatform.ts

```diff
@@ -184,7 +184,7 @@ export function withExtendedResolver(
   const defaultResolver = metroResolver.resolve;
   const resolver = isFastResolverEnabled
     ? createFastResolver({
-        preserveSymlinks: config.resolver?.unstable_enableSymlinks ?? true,
+        preserveSymlinks: true,
```

Comment:
Starting from Metro 0.81 it's no longer possible to turn off Symlink support.

https://github.com/facebook/metro/commit/e52b5ffb24d0388c26441457bb695b53f44a6960